Formula/metabase.rb

```diff
@@ -21,6 +21,8 @@ class Metabase < Formula
   def install
     if build.head?
       Language::Node.setup_npm_environment
+      # help yarn find npm's node-gyp binary inside superenv
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/lib/node_modules/npm/bin/node-gyp-bin"
```

Comment:
> also impossible these days with the sandbox in place

But the sandbox is only active when executing brew commands. A user with the current yarn formula executing `yarn global add webpack` will create a symlink in `HOMEBREW_PREFIX/Cellar/node/8.2.1/bin/webpack`.